styles.css

```diff
@@ -0,0 +1,663 @@
+    float: right;
+    align-items: center;
+  }
+  nav {
+    margin: 0;
+  }
+  .dog-profiles section {
+    width: 75%;
+  }
+  .home-container img {
+    width: 70%;
+  }
+  .see-more {
+    width: 95%;
+    text-align: left;
+    padding: 1.5rem 0 1.5rem 0;
+  }
+  .blog-preview {
+    border-top: 1px dotted #ccc;
+    border-left: none;
+    width: 95%;
+    padding-top: 3rem;
+    margin: 1rem;
+  }
+  .blog-preview p {
+    visibility: hidden;
+    height: 0;
+  }
+  .blog-article {
+    width: 100%;
+  }
+  .blog-article img {
+    width: 70%;
+    height: 70%;
+    align-items: center;
+  }
+
+  .blog-text {
+    width: 95%;
+    text-align: left;
+  }
+
+  footer div img {
+    width: 60%;
+  }
+  footer p {
+    width: 95%;
+    padding-left: 6rem;
+  }
+  footer nav {
+    width: 60%;
+    padding-right: 0rem;
+    padding-left: 5rem;
+  }
+}
+
+@media screen and (min-width: 577px) and (max-width: 767px) {
+  .container {
+    width: 95%;
+    margin: 1rem;
+  }
+  .header-logo {
+    width: 60%;
+    position: absolute;
+    top: 15%;
+    left: 4%;
+  }
+  .header h1 {
+    font-size: 0rem;
+  }
+  .header-checkout {
+    top: 20%;
+    left: 2%;
+    width: 20%;
+  }
+  .dog-profiles section {
+    width: 60%;
+  }
+  .see-more {
+    width: 100%;
+    text-align: left;
+    padding: 1.5rem 0 1.5rem 0;
+  }
+  .blog-preview {
+    border-top: 1px dotted #ccc;
+    border-left: none;
+    width: 95%;
+    padding-top: 3rem;
+    margin: 1rem;
+  }
+  .blog-preview p {
+    visibility: hidden;
+    height: 0;
+  }
+  .footer-container {
+    width: 100%;
+    max-height: 30%;
+    float: left;
+    clear: both;
+    background-color: #103861;
+    text-align: center;
+    padding: 2rem;
+    color: #fff;
+  }
+}
+@media screen and (min-width: 768px) and (max-width: 900px) {
+  .container {
+    width: 60%;
+    float: left;
+  }
+  .dog-profiles section {
+    width: 40%;
+  }
+  .header-logo {
+    text-align: left;
+    position: absolute;
+    top: 10%;
+    left: 5%;
+    float: left;
+    clear: both;
+  }
+  .see-more {
+    width: 75%;
+  }
+  .blog-preview {
+    display: flex;
+    flex-flow: column wrap;
+    border-top: none;
+    border-left: 1px dotted #ccc;
+    padding-left: 1rem;
+    width: 25%;
+    float: right;
+    margin: 2rem;
+  }
+  .footer-container {
+    width: 100%;
+    max-height: 30%;
+    float: left;
+    clear: both;
+    background-color: #103861;
+    text-align: center;
+    padding: 2rem;
+    color: #fff;
+  }
+}
+@media screen and (min-width: 901px) {
+  .container {
+    width: 60%;
+    float: left;
+  }
+  .dog-profiles section {
+    width: 25%;
+  }
+  .header-logo {
+    text-align: left;
+    position: absolute;
+    top: 10%;
+    left: 5%;
+    float: left;
+    clear: both;
+  }
+  .blog-preview {
+    display: flex;
+    flex-flow: column wrap;
+    border-left: 1px;
+    padding-left: 1rem;
+    padding-right: 1rem;
+    width: 25%;
+    float: right;
+    margin: 3rem 0 0 4rem;
```

Comment:
It looks like you are pushing the element off the page using margin. Instead use display:none; to actually hide it and make it take no space on the page.

Reply:
Got it. Adjusted.